merino/configs/default.toml

```
@@ -79,3 +79,10 @@ backend = "remote-settings"
 cron_interval_sec = 60
 resync_interval_sec = 10800
 score = 0.3
+
+[default.providers.top_pick]
```

To be consistent, let's use "Top Picks" in Merino.
 
```suggestion
[default.providers.top_picks]
```